packages/fxa-profile-server/lib/routes/_core_profile.js

```diff
@@ -85,6 +86,9 @@ module.exports = {
       if (typeof body.authenticatorAssuranceLevel !== 'undefined') {
         result.twoFactorAuthentication = body.authenticatorAssuranceLevel >= 2;
       }
+      if (typeof body.subscriptions !== 'undefined') {
```

Comment:
Can we only return this if an appropriate scope is set? It makes me nervous that any RP can get a list of all the subscriptions that are set for a relier.

If we guard the subscription list, we will need a way to surface whether the user has *any* active subscriptions to other RPs, e.g., SUMO doesn't care about the actual subscription list for a user, they just want to know whether the user has at least one active subscription.

Reply:
> It makes me nervous that any RP can get a list of all the subscriptions that are set for a relier.

Well, each RP should only be shown the subset of subscription capabilities listed for them in `clientCapabilities` in the server config. That's meant to guard visibility of subscription status.

> we will need a way to surface whether the user has any active subscriptions to other RPs

It occurred to me today that I might add a few "special" products like `default` that every registered user gets regardless of subscriptions and `defaultSubscribed` that users with at least one subscription to any product gets. Then, we should be able to list those in `clientCapabilities` for a client like SUMO.

Reply:
I added the special products for any registered user and any user with a subscription. Also, I think the restricted capability filtering by client ID takes care of guarding subscriptions visibility here.